File: weak17/second_qa/prime.py
def prime(number):
    if number <= 1:
        return False 
    if number == 2 :
        return True 
    if number % 2 == 0 :
        return False 
    for i in range(3,number,2):
        if number % i == 0 :
            return False 
    return True 
print(prime(13))

File: weak18/programs/prime.js
let number = parseInt(prompt("Enter a number:"));
let isPrime = true;

if (number <= 1) {
    isPrime = false;
} else {
    for (let i = 2; i <= Math.sqrt(number); i++) {
        if (number % i === 0) {
            isPrime = false;
            break;
        }
    }
}

if (isPrime) {
    console.log(number + " is a Prime Number.");
} else {
    console.log(number + " is NOT a Prime Number.");
}
